utils/http_test.go

```diff
@@ -66,3 +66,39 @@ func TestUnitGetPractitionerIDFromVars(t *testing.T) {
 		So(PractitionerID, ShouldBeEmpty)
 	})
 }
+
+func TestUnitGetAttachmentIDFromVars(t *testing.T) {
+	Convey("Get Attachment ID", t, func() {
+		vars := map[string]string{
+			"attachment_id": "67890",
+		}
+		PractitionerID := GetAttachmentIDFromVars(vars)
+		So(PractitionerID, ShouldEqual, "67890")
```

Comment:
Should the variable be called `attachmentID`?

```suggestion
                attachmentID := GetAttachmentIDFromVars(vars)
		So(attachmentID, ShouldEqual, "67890")
```